matrix_messages.rst

```diff
@@ -0,0 +1,92 @@
+Raiden transport messages
```

Comment:
Shouldn't these go to the already existing `transport.rst` ?

Reply:
yea, I can move it there if we agree on message format. On the other hand transport document should maybe only contain low-level transport specification and messages SPEC may be a separate document.